ag/ag.proto

```diff
@@ -259,9 +260,9 @@ message LabRequest {
     bool groupLabs = 4;
 }
 
-// messages of type Void used when we want a boolean answer from the server
+// Messages of type Void used when we want a boolean answer from the server
 // GRPC status code 0 indicates that the requested action was a success
-// any other status will include a message explaining what went wrong
+// any other status will include a message explaining what went wrong.
 message Void {}
```

Comment:
What about:
_Void contains no fields. Void still contains a gRPC status code, which can be checked for success or failure. Status code 0 indicates that the requested action was successful, whereas any other status code indicates some failure. As such, the status code can be used as a boolean result from the server._

Reply:
Done.